core/src/main/java/software/amazon/awssdk/http/DefaultErrorResponseHandler.java

```diff
@@ -111,7 +111,7 @@ private String contentToString(InputStream content, String idString) throws Exce
         try {
             return IoUtils.toString(content);
         } catch (Exception e) {
-            LOG.info(String.format("Unable to read input stream to string (%s)", idString), e);
+            log.info(String.format("Unable to read input stream to string (%s)", idString), e);
```

Comment:
Any difference in performance between using {} to substitute values and this current form?